README.md

```diff
@@ -21,17 +21,35 @@ This library uses the ORS API for request validation. To understand the input of
 
 ## Developement Setup
 
-Clone the openrouteservice-js package from GitHub into a developement environment of your choice.
+### WSL Setup
 
-Make sure you have installed the npm, node.js and nvm package managers to ensure you will be able to work in the repository.
+Download and install an Ubuntu terminal environment.
+Set it as the default distribution in the command line ``wsl.exe --set-default [Distro]``
+You can use ``wsl.exe -l`` to check which distribution is currently the default one.
+
+You might need to update the apt package manager to access and be able to install packages. You can do this with the command ```sudo apt update && sudo apt upgrade```
+
+*Recommended steps (but not required):*
+Install nano and GitHub credential manager.
+
+### Openrouteservice Package Setup
+
+Clone the openrouteservice-js repository from GitHub into a developement environment of your choice.
+Install Git Bash if you do not have it already.
+
+Set the terminal to wsl.exe and make sure you have installed the npm, node.js and nvm package managers to ensure you will be able to work in the repository.
+
+Install the dependencies with ```npm install```
+
```

Comment:
This section should be moved down above the "Running Tests" section.
I think a header of 'Development setup' is a bit more fitting.

You can add a Prerequisites or Requirements list [like here](https://github.com/GIScience/openrouteservice-app#install-dependencies-locally)
The note to the notes on Git Bash and wsl can be removed, as they also are not specific to this repository.